OWNERS

```diff
@@ -10,6 +10,7 @@ reviewers:
   - martinkennelly
   - npinaeva
   - pperiyasamy
+  - pliurh
```

Comment:
hi @pliurh its not this list that we use for reviewers unfortunately.. the system is different. I'll add you manually.
closing PR.
eventually @ovn-kubernetes/ovn-kubernetes-admin we need to clean this up